components/gif_picker/components/SearchGrid/SearchGrid.tsx

```diff
@@ -162,8 +304,8 @@ export class SearchGrid extends PureComponent {
         //const itemWidth = this.numberOfColumns === NUMBER_OF_COLUMNS_PORTRAIT ? 100 / NUMBER_OF_COLUMNS_PORTRAIT : 100 / this.numberOfColumns;
```

Comment:
since we are already improving the code, can you remove this commented code? it seems unnecessary